std-bits/tableau/src/main/java/org/enso/tableau/HyperReader.java

```diff
@@ -104,6 +107,30 @@ private static HyperProcess getProcess() throws IOException {
     return process;
   }
 
+  private static final class TableauClassLoader extends ClassLoader {
+    private TableauClassLoader() {
+      super(HyperReader.class.getClassLoader());
+    }
+
+    @Override
+    public InputStream getResourceAsStream(String name) {
```

Comment:
What is the caller stack trace? What the tableu code is trying to achieve? Why does it need `InputStream`? Can we prevent it to ask for input stream?

Reply:
We can't prevent it. It's part of hyper initialization.

Reply:
My [investigation](https://github.com/enso-org/enso/pull/12204#discussion_r1940612856) suggests there is a way to prevent the call.